AUTHORS

```diff
@@ -3460,5 +3460,6 @@ liuxingbaoyu <30521560+liuxingbaoyu@users.noreply.github.com>
 Sergey Nazaryev <sergey@nazaryev.com>
 William Marlow <william.marlow@ibm.com>
 Keyhan Vakil <60900335+airtable-keyhanvakil@users.noreply.github.com>
+Feng Yu <F3n67u@outlook.com>
```

Comment:
This line is a side-effect of running `tools/update-authors.js` after updating `.mailmap`. I think it's fine to leave it, rather than removing it only so that the scheduled job re-adds it in a few days.